### ISSUE TITLE
Build failing on Ubuntu 16.04

### ISSUE THREAD

**Issue by rgooler** [chef-cookbooks/fail2ban#45]
### Cookbook version
3.1.0

### Chef-client version
[Version of chef-client in your environment]

### Platform Details
Ubuntu 16.04

### Scenario:
0.9.3-1

### Steps to Reproduce:
Install cookbook as is - fails to start fail2ban

### Expected Result:
cookbook works on current version of ubuntu

### Actual Result:
cookbook doesn't work

**Comment by fletchowns** [comment]
I recently started getting a failure to start fail2ban service on initial converge as well, but on RedHat 7.2.

Doesn't seem like anything changed recently with this cookbook but the machines it was failing on have fail2ban 0.9.6, where my older machines that have 0.9.2 are not failing.

**Comment by fletchowns** [comment]
Maybe my issue is different...

```
==> /var/log/messages <==
Feb 22 20:20:56 ip-10-0-38-103 systemd: Starting Fail2Ban Service...
Feb 22 20:20:56 ip-10-0-38-103 fail2ban-client: ERROR  There is no directory /var/run/fail2ban to contain the socket file /var/run/fail2ban/fail2ban.sock.
Feb 22 20:20:56 ip-10-0-38-103 systemd: fail2ban.service: control process exited, code=exited status=255
```

**Comment by iennae** [comment]
This should be fixed on Ubuntu 16.04 now with release 4.0.0. Please take a look and reopen if you are still having issues.
